configure.ac

```diff
@@ -8,6 +8,8 @@ define(_CLIENT_VERSION_IS_RELEASE, false)
 define(_COPYRIGHT_YEAR, 2022)
 define(_COPYRIGHT_HOLDERS,[The %s developers])
 define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[[Bitcoin Core and Dogecoin Core]])
+define(_CURRENT_RELEASE_DATE,[[Jan 01 1970]])
+define(_NEXT_RELEASE_DURATION,[180])
```

Comment:
Since we didn't meet a 180 day release cycle the last release (and likely won't now either) I'd suggest putting this at 270 for the time being (as we met that the last 4 releases) and we can then lower it to 180 when we have a better track record of making that happen?

i.e. I'd like this to reflect reality instead of intent, but am open to other opinions.

Reply:
Works for me. We can change this at every release date to bring it more into line with reality.

Reply:
Agree. Although we can change our minds up and until release commit, I do think that release pressure cooker is not the right moment to have a discussion about the duration, so it's better to pre-sort up front and prevent mistakes under stress.